kasa/cli.py

```diff
@@ -911,11 +911,9 @@ async def effect(dev: Device, ctx, effect):
         echo("Device does not support effects")
         return
     if effect is None:
-        raise click.BadArgumentUsage(
-            "Setting an effect requires a named built-in effect: "
-            + f"{light_effect.effect_list}",
-            ctx,
-        )
+        echo(f"Light effect: {light_effect.effect}")
+        return light_effect.effect
```

Comment:
This should probably also list the available effects?

Reply:
Done